### PR TITLE
chore: Prevent Microsoft MUAs from sending reactions

### PR DESCRIPTION
**Pull request by mpldr**
# Description

Microsoft added a way to "react" to emails which is turned into
> [Image] <Name> liked this message.

No MUA except Microsoft's latest ones displays these properly and they are not providing significant value to an Email-exchange.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary, I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] **I have added information about changes made in this PR to `CHANGELOG.md`**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +20,7 @@ All notable changes to this project will be documented in this file. The format
 
 - **Fail2ban**:
   - Bump version to [1.1.0](https://github.com/fail2ban/fail2ban/releases/tag/1.1.0). For more information, check the [changelog](https://github.com/fail2ban/fail2ban/blob/1.1.0/ChangeLog).
+  - Disable Microsoft reactions to outgoing mail.
```
Comment (polarathene) [inline CHANGELOG.md:23]: Please add a reference link to PR:

```suggestion
  - Disable Microsoft reactions to outgoing mail ([#4120](https://github.com/docker-mailserver/docker-mailserver/pull/4120))
```

**Review by polarathene — CHANGES_REQUESTED** [review]

**Review by casperklein — DISMISSED** [review]
I am ambivalent about this PR. Personally I don't like that MS feature. But I am also not a fan of including app specific hacks into DMS.

For me, it would be fine to do such things via `user-patches.sh`.

**Comment by mpldr** [comment]
> But I am also not a fan of including app specific hacks into DMS.

Me neither, but if a multi-billion dollar corporation feels like undermining standards, we – as in people invested in keeping the technology open and accessible – should make sure that this does not work with our systems. Google pulled something similar with XMPP and I would find it sad, if Microsoft did it to Email.

**Review by polarathene — APPROVED** [review]

**Comment by polarathene** [comment]
We do not have the permission to rebase or merge master to update your branch, you'll need to update your branch again. Conflict would be from a recent PR that updated the `CHANGELOG.md`.

If the PR here doesn't cause any issues for users (_I don't know if it makes sense to expect the feature compatibility with DMS_), merging is fine. Otherwise I'd agree with @casperklein and just add a tip in our docs about it with `user-patches.sh`.

**Review by polarathene — APPROVED** [review]

**Pull request merged**